pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -849,6 +863,14 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceExpectedMountsPolicy(containerI
 	return hooks.AddOCIHook(spec, hooks.CreateRuntime, hook)
 }
 
+func (pe *StandardSecurityPolicyEnforcer) EnforceLogging() error {
+	if !pe.LoggingEnabled {
+		logrus.Debug("Logging is disabled in the policy")
+		logrus.SetOutput(io.Discard)
```

Comment:
I dont know what the security model is, but this may not disable logging.
The logger may have hooks that write info (similar to the `etwlogrus`), so you want to: 
```go
logrus.StandardLogger().Hooks = nil
```
Though the gcs may also create their own logger (via `	logrus.New()`)....

Can we enforce the policy on the host side [here](https://github.com/microsoft/hcsshim/blob/bf98c3da3c4385f8f30468847146898891f044c3/internal/uvm/create_lcow.go#L726-L728)?

Reply:
No, the host is untrusted. All policy must be enforced on the guest side.